huaweicloud/services/secmaster/resource_huaweicloud_secmaster_incident.go

```diff
@@ -0,0 +1,595 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product SecMaster
+// ---------------------------------------------------------------
+
+package secmaster
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceIncident() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIncidentCreate,
+		UpdateContext: resourceIncidentUpdate,
+		ReadContext:   resourceIncidentRead,
+		DeleteContext: resourceIncidentDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceIncidentImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"workspace_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
```

Comment:
missing description.
and please check the region parameter.

Reply:
fixed